_pages/en/strategy-build-vs-buy.md

```diff
@@ -44,18 +48,18 @@ permalink: /strategy-build-vs-buy.html
 
 The strategy includes:
 
-1. A **guiding policy**, which serves to clarify the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when obtaining new or enhancing existing IT Solutions in Employment and Social Development Canada (ESDC).
-2. A **coherent set of actions** (an action plan), which serve to operationalize the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when obtaining new or enhancing existing IT Solutions in ESDC.
+1. A **guiding policy**, which serves to clarify the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and on [Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when acquiring new or enhancing existing IT Solutions in Employment and Social Development Canada (ESDC).
+2. A **coherent set of actions** (an action plan), which serve to operationalize the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and on [Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when acquiring new or enhancing existing IT Solutions in ESDC.
 
-The intent behind this strategy is to communicate a decision by the Chief Information Officer (CIO) (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+The intent behind this strategy is to communicate a decision by the ESDC Chief Information Officer (CIO) (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
 
 ### Targeted Stakeholders
 
-This **strategy document** is targeted to stakeholders involved in obtaining an IT Solution as an outcome of an initiative as defined in the Policy on Project and Programme Management (PPPM).
+This **strategy document** is targeted to stakeholders involved in acquiring an IT Solution as an outcome of an initiative as defined in the Policy on Project and Programme Management (PPPM).
```

Comment:
suggest rewording to "... an IT Solution to help fulfill outcomes of an IT-enabled initiative as defined in a Project or Programme Business Case"

The rationale is that an IT Solution should not be seen as a business outcomes. We are thriving to have lines of businesses better define what they are striving to achieve with IT. This is expected to be defined in a business case (via outcomes and metrics) as per the Directive on Programme Management and the Directive on Project Management (both instruments under the PPPM).

Reply:
Good point, will adjust!